index.html

```diff
@@ -12,21 +12,52 @@
     <div id="signin_background">
         <h1>Sign in</h1>
 
-        <form class="info-user">
-            <label for="username">Username:</label>
+    <div id="signin-form">
+        <h1>Connection</h1>
+        <form class="InfoUser">
+            <label for="username">Nom d'utilisateur:</label>
             <br>
-            <input type="text" id="username" name="username" placeholder="Enter Username:"><br><br>
+            <input type="text" id="signin-username" name="signin-username"
+                placeholder="Entrez nom d'utilisateur:"><br><br>
 
-            <label for="password">Password:</label>
+            <label for="signin-password">Mot de passe:</label>
             <br>
-            <input type="password" id="password" name="password" placeholder="Enter password:"><br><br>
+            <input type="password" id="signin-password" name="signin-password"
+                placeholder="Entrez mot de passe:"><br><br>
```

Comment:
Ne pas utiliser de \<br>!
```suggestion
                placeholder="Entrez mot de passe:">
```